pennylane_qiskit/converter.py

```diff
@@ -59,19 +61,33 @@ def _check_circuit_and_bind_parameters(quantum_circuit: QuantumCircuit, params:
         params (dict): dictionary of the parameters in the circuit
 
     Returns:
-        qc (QuantumCircuit): quantum circuit with bound parameters
+        QuantumCircuit: quantum circuit with bound parameters
+        dict[qiskit.circuit.Parameter, pennylane.variable.VariableRef]: a dictionary mapping
+            qiskit parameters to PennyLane variables
+
     """
     if not isinstance(quantum_circuit, QuantumCircuit):
         raise ValueError("The circuit {} is not a valid Qiskit QuantumCircuit.".format(quantum_circuit))
 
     if params is None:
-        return quantum_circuit
+        return quantum_circuit, {}
+
+    # map qiskit parameters to PennyLane differentiable VariableRefs.
+    var_ref_map = {}
 
     for k, v in params.items():
-        if isinstance(v, qml.variable.Variable):
-            params.update({k: v.val})
+        # iterate through the parameters
+        if isinstance(v, qml.variable.VariableRef):
+            # If a parameter is a Variable reference, then it represents
+            # a differentiable parameter.
```

Comment:
This could go as a dictionary comprehension as well (though might be too long of a one-liner):
```python
var_ref_map = {k:v for k, v in params.items() if isinstance(v, qml.variable.VariableRef)}
```